backend/python/tapa/tapac.py

```diff
@@ -229,13 +229,35 @@ def create_parser() -> argparse.ArgumentParser:
            'The key is the region name, the value is a list of modules.'
            'Replace the outdated --directive option.'
   )
-  group.add_argument(
+
+  strategies = parser.add_argument_group(
+      title='Strategy',
+      description=('Choose different strategy in floorplanning and codegen '
+                   '(advanced usage).'),
+  )
```

Comment:
Why is this a separate argument group from floorplanning?

Reply:
split the basic configurations from the DSE options